app/default/content_routes.py

```diff
@@ -54,3 +57,24 @@ def contact_us():
 def cookie_policy():
     current_app.logger.info("Cookie policy page loaded.")
     return render_template("cookie_policy.html")
+
+
+@content_bp.route("/privacy", methods=["GET"])
+def privacy():
+    current_app.logger.info("Privacy_notice page loaded.")
+    round_data = get_round_data_fail_gracefully(
+        Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID
+    )
```

Comment:
I think we're moving to support multi-funds, it would be good to parametrize these instead of assuming it's COF from config. 

I.e see: https://github.com/communitiesuk/funding-service-design-frontend/pull/302

https://github.com/communitiesuk/funding-service-design-frontend/blob/447cba73d2e3f42e759b3b2f74546db684f4ac42/app/default/routes.py#L26-L27

Reply:
CC: @gio-karunakaran @srh-sloan I believe you paired on this, so may be interested in this!